deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -52,9 +52,13 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
-      {{- if $useServiceAccount }}
```

Comment:
Not sure about this, it could introduce a breaking change , we need to find a way that won't break existing users scripts that may rely on this existing HELM setup

Reply:
It's not a breaking change because the existing logic is respected. It replaces this `if` with another one but still considering `$useServiceAccount`.